src/components/RiddleCard.tsx

```diff
@@ -1,36 +1,121 @@
-import { Card, CardContent, CardHeader, CardMedia } from '@mui/material';
+import {
+	Box,
+	Button,
+	Card,
+	CardActions,
+	CardContent,
+	CardHeader,
+	CardMedia,
+	Chip,
+	Container,
+	Grid,
+	Stack,
+	Typography
+} from '@mui/material';
 import {
 	CancelRounded,
 	CheckCircleOutlineRounded,
 	HelpOutlineRounded
 } from '@mui/icons-material';
+import { CircleFlag } from 'react-circle-flags';
+import LensIcon from '@mui/icons-material/Lens';
+import { FC, useMemo } from 'react';
+
+import { CountryCode } from '../utils/CountryCodes.ts';
```

Comment:
Není nutné importovat i s koncovkou `.ts` nebo `.tsx`.

Reply:
Dobrý postřeh, import se vložit automaticky - nevíš o nějakém lintovacím pravidle, které by to podchytilo a fixnulo?

Reply:
Popravdě nevím. Ve VS code (a ani předtím s JetBrains) jsem se s tí mnesetkal, vždycky mi to automaticky importovalo bez koncovek.